react/components/RenderProvider.tsx

```diff
@@ -80,6 +84,15 @@ class RenderProvider extends Component<Props, RenderProviderState> {
     runtime: PropTypes.object,
   }
 
+  public sendInfoFromIframe = debounce(() => {
+    const context = this.getChildContext()
+    if (window.top !== window.self) {
```

Comment:
For the future: we should have an easier `isIframe` function available

Reply:
Oh, and we should also check for the existence of `window.top.__provideRuntime`

Reply:
And we could probably only incur in the `getChildContext` call after we're sure we're sending it.